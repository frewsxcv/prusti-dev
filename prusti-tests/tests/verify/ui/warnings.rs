#![allow(dead_code)]
#![allow(unused)]
use prusti_contracts::*;

#[requires(result == 0)]
#[ensures(result == 0)]
fn foo(result: u32) -> u32 {
    0
}

#[trusted]
fn main() {}
