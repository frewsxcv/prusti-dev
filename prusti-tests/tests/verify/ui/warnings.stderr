error: it is not allowed to use the keyword `result` as a function argument
 --> $DIR/warnings.rs:7:8
  |
7 | fn foo(result: u32) -> u32 {
  |        ^^^^^^^^^^^

error: aborting due to previous error

